src/Components/3DV/SceneView.tsx

```diff
@@ -344,7 +346,7 @@ function SceneView(props: ISceneViewProps, ref) {
                     const es = someMeshFromTheArrayOfMeshes.getBoundingInfo()
                         .boundingBox.extendSize;
                     const es_scaled = es.scale(
-                        zoomMeshIds && zoomMeshIds.length < 10 ? 5 : 3
+                        zoomMeshIds && zoomMeshIds.length > 10 ? 5 : 3
```

Comment:
why is 10 the magic number here?

Reply:
Updated code and added a comment. What we actually want to do here is set the scale to be a little further out if we are zooming into an element vs the whole model.